### DISCUSSION TITLE
mitiq.dev needs re-verification

### DISCUSSION THREAD

**Discussion by Misty-W** [unitaryfund/mitiq#1943] [General]
Received this email notification from GitHub a few minutes ago:

We can no longer verify your ownership of mitiq.dev
 
We are unable to find a verification TXT record on mitiq.dev. Please [verify](https://github.com/organizations/unitaryfund/settings/pages_verified_domains/mitiq.dev) your domain within 7 days by following our instructions before it becomes unverified.

When a domain is unverified, other users can freely publish GitHub Pages sites on that domain. Additionally, if another user verifies this domain, your GitHub Pages sites on that domain will move back to [github.io](http://github.io/).

**Comment by natestemen** [comment]
This is taken care of now! Thanks for raising.

https://github.com/organizations/unitaryfund/settings/pages
